lxd/storage/backend_lxd.go

```diff
@@ -1826,8 +1827,8 @@ func (b *lxdBackend) imageConversionFiller(imgPath string, imgFormat string) fun
 
 		// Convert uploaded image from backups directory into RAW format on the instance volume.
 		cmd := []string{
-			// Use prlimit to limit QEMU to 1 GiB address space and 120 seconds of CPU time.
-			"prlimit", "--cpu=120", "--as=1073741824",
+			// Use prlimit to limit QEMU to 1 GiB address space and 5 minutes of CPU time.
```

Comment:
this has been bumped but the commit message nor comment explains why.

Reply:
Will add the comment. The reason is that large VMs such as Windows can take longer then 2 minutes to convert.